src/components/SearchBar.tsx

```diff
@@ -326,8 +311,8 @@ export function SearchBar({
     return autocompleteResponse?.results.map((result, i) => (
       <Fragment key={i}>
         <DropdownItem
-          className={cssClasses.optionContainer}
-          focusedClassName={classNames(cssClasses.optionContainer, cssClasses.focusedOption)}
+          className='flex items-stretch py-1.5 px-3.5 cursor-pointer hover:bg-gray-100'
+          focusedClassName={classNames('flex items-stretch py-1.5 px-3.5 cursor-pointer hover:bg-gray-100', cssClasses.focusedOption)}
```

Comment:
another classNames usage here

Reply:
Should I keep the one in const definition of activeClassName on line 370 since there's some logic being used there?

Reply:
Ohh yeah sure that one's a little weird, we can have whoever takes the item figure out how to deal with it properly